src/App.tsx

```diff
@@ -6,10 +6,9 @@ import { fab } from '@fortawesome/free-brands-svg-icons'
 import { fas } from '@fortawesome/free-solid-svg-icons'
 import { far } from '@fortawesome/free-regular-svg-icons'
 import { TokenContextProvider } from './context/token'
-import { ErrorContextProvider } from './context/error'
 import { ProfileContextProvider } from './context/profile'
 import Header from './components/Header'
-import Errors from './components/Errors'
+import MessageFlash from './components/FlashMessages'
```

Comment:
Why renaming from FlashMessages to MessageFlash? Keep it the same as the component